controllers/SiteController.php

```diff
@@ -308,26 +294,103 @@ public function actionChangeLanguage($lang)
         }
     }
 
-    public function actionDesignList()
-    {
-        return $this->render('design-list');
+    public function actionDesignList($viewMode = NULL) {
+        if (!empty($viewMode)) {
+            $viewMode = 1;
+        } else {
+            $viewMode = 0;
+        }
+        $query = new Query;
+        $query->select(['moqup.*', 'user.username as username'])
+                ->from('moqup')
+                ->where(['!=', 'user_id', Yii::$app->user->id])
+                ->leftJoin('user', 'moqup.user_id = user.id')
```

Comment:
@sapkaljagruti Here you may use model relations. [Reference.](https://www.yiiframework.com/doc/guide/2.0/en/db-active-record#relational-data)